src/main/java/no/sikt/sws/IndexHandler.java

```diff
@@ -33,11 +34,19 @@ Pair<String, Integer> inputToOutputWithStatus(String body, RequestInfo request,
         try {
             var url = workspace + "-" + resourceIdentifier;
             logger.info("URL: " + url);
-            var response = openSearchClient.sendRequest(httpMethod, url, body);
+
+            var response = openSearchClient.sendRequest(
+                    httpMethod,
+                    url,
+                    WorkspaceStripper.add(body, workspace, resourceIdentifier));
             logger.info("response-code:" + response.getStatus());
             logger.info("response-body:" + response.getBody());
 
-            return Pair.of(response.getBody(), response.getStatus());
+
+            var responseBody = WorkspaceStripper.remove(response.getBody(), workspace);
+            logger.info("response-body:" + responseBody);
```

Comment:
Can you also log the body before the strip? usefull for debugging later

Reply:
`            logger.info("response-code:" + response.getStatus());
            logger.info("response-body:" + response.getBody());


            var responseBody = WorkspaceStripper.remove(response.getBody(), workspace);
            logger.info("response-body:" + responseBody);`
            
            Skjønner ikke det, koden min ser slik ut.